### ISSUE TITLE
SOLR

### ISSUE THREAD

**Issue by andyfil** [andyfil/ProyectoWebIR#1]
Explicar al resto como utilizarlo en windows

**Comment by jbonanat** [comment]
Ya lo expliqué por mail, si querés pongo por acá como sería...

**Comment by andyfil** [comment]
Como quieras. Yo la agregue para que quede registrado. SI queres explicate como seria agregar el core y masomenos que es

**Comment by jbonanat** [comment]
Sí, mañana o pasado creo la carpeta de configuración de Solr y le agrego un readme.

**Comment by jbonanat** [comment]
Ahí les muestro cómo definir los distintos archivos de configuración. Voy a sacarlos de los ejemplos y borrarles basura así queda mejor.

**Comment by andyfil** [comment]
Buenisimo!
